api/valor_api/backend/core/evaluation.py

```diff
@@ -434,14 +405,11 @@ def _fetch_evaluation_from_subrequest(
 def create_or_get_evaluations(
     db: Session,
     job_request: schemas.EvaluationRequest,
-) -> tuple[
-    list[schemas.EvaluationResponse],
-    list[schemas.EvaluationResponse],
-]:
+) -> list[schemas.EvaluationResponse]:
     """
     Creates evaluations from evaluation request.
 
-    If an evaluation already exists, it will be returned as running.
+    If an evaluation already exists, it will be returned with a non-pending status.
```

Comment:
can we change "non-pending status" to the actual status it returns?

Reply:
This is saying that existing evaluations could have any status other than pending

Reply:
right, but the way it's worded seems unnecessarily vague / non-descriptive. consider this suggestion a nit

Reply:
was this addressed in the latest changes? maybe I missed it

Reply:
> can we change "non-pending status" to the actual status it returns?

If an evaluation already exists, it will be returned with a non-pending status.
If an evaluation already exists, it will be returned with a running, done, failed or deleting status.

I prefer the original as it feel it is more concise.